src/platform-integration/web/wasm.md

```diff
@@ -94,52 +109,59 @@ Server started on port 8080
 
 #### Load it in a browser
 
-As of {{page.last-update}}, there are two known browser types that should run
-Flutter/Wasm content.
+As of {{page.last-update}},
+there are two known browser types that should run
+Flutter/Wasm content:
 
 - Chromium-based browsers
-  - Run version 113 or greater.
-  - Set the `enable-webassembly-garbage-collection`
-    [flag](https://developer.chrome.com/docs/web-platform/chrome-flags/).
+  - Version 119 or greater.
 - Firefox
-  - Run the
-    [nightly channel](https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly)
-    build. (Verified on v.115.0a1)
-  - Set two additional preferences in
-    [about:config](https://support.mozilla.org/kb/about-config-editor-firefox):
-    - `javascript.options.wasm_function_references`
-    - `javascript.options.wasm_gc`
+  - Version 120 or greater.
+    This currently includes the beta, nightly, and developer
```

Comment:
**nit:** You should remove `currently`. All of our docs should be considered current, so you don't need this modifier.

Reply:
Good point.

This will change very soon or could change at any time, so I tried a different wording. Let me know what you think.